app/enc/geo/location.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2022 The Drivers Cooperative. All rights reserved.
+// Use of this source code is governed by a dual
+// license that can be found in the LICENSE file.
+
+package geo
+
+import (
+	"fmt"
+	"unsafe"
+)
+
+type Location struct {
+	Lat float64 // latitude
+	Lon float64 // longitude
+}
+
+type LocationWKB struct {
```

Comment:
what WKB stand for?
is non-intuitive for me, maybe a sign for adding documentary comment.

Reply:
https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry